.eslintrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+    parser: '@typescript-eslint/parser',
+    plugins: ['@typescript-eslint'],
+    root: true,
+  };
```

Comment:
ESLint supports multiple configuration file formats:

https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-file-formats

But it only recognizes a single configuration file per directory:

https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-file-formats:~:text=If%20there%20are%20multiple%20configuration%20files%20in%20the%20same%20directory

So the presence of this file would cause `.eslintrc.yml` to be ignored. What needs to be done is the configuration from this `.eslintrc.cjs` file should be merged in to `.eslintrc.yml` and then the `.eslintrc.cjs` file deleted.

I realize this confusion is the result of [the low quality installation instructions](https://github.com/per1234/.github/blob/main/workflow-templates/check-typescript-task.md) I wrote for the template. I have been trying to get this template brought up to the Tooling Team standards and migrated to the `arduino/tooling-project-assets` repo, but haven't made too much progress on that yet.